java/events.md

```diff
@@ -2,76 +2,78 @@
 
 ## What is a Smart Contract Event
 
-Events can be emitted by all smart contract functions that are triggered by a transaction, and are an important piece of the Ethereum application architecture puzzle.
+All smart contract functions triggered by transactions emit events, and are an important piece of the Ethereum application architecture puzzle.
```

Comment:
Smart contract functions that are triggered by transactions CAN emit events, but not all of them do, so this sentence needs to be modified a bit.

Reply:
Hows about this change?

Reply:
Add a 'they' to "...and `they` are an important piece..." and we've got a deal! :D

Reply:
So many words!